src/adapter/debugAdapter.ts

```diff
@@ -173,7 +173,37 @@ export class DebugAdapter implements IDisposable {
     this._pauseOnExceptionsState = 'none';
     if (params.filters.includes('caught')) this._pauseOnExceptionsState = 'all';
     else if (params.filters.includes('uncaught')) this._pauseOnExceptionsState = 'uncaught';
-    if (this._thread) await this._thread.setPauseOnExceptionsState(this._pauseOnExceptionsState);
+    if (this._thread) {
+      const result = this._thread.setPauseOnExceptionsState(this._pauseOnExceptionsState);
+      const logger = this._services.get<ILogger>(ILogger);
+      if (this._configurationDoneDeferred.hasSettled()) {
+        logger.verbose(LogTag.Internal, 'setExceptionBreakpoints: awaiting blocking response');
+        await result;
+      } else {
+        result
+          .then(() => {
+            logger.verbose(
+              LogTag.Internal,
+              'setExceptionBreakpoints: succesful non-blocking response',
+            );
+          })
```

Comment:
We get this info from the CDP response in the log (setPauseOnExceptionsState is a one-line function that runs the CDP call) method don't think we need extra stuff here.